dashboard/lti_new.py

```diff
@@ -24,12 +24,23 @@
 DUMMY_CACHE = 'DummyCache'
 
 
+class LTIError:
```

Comment:
When I suggested moving this error message to a new class, I had in mind that the new class would extend `Exception` and it could be invoked with a `raise e` call.  If it's working the way it's currently coded, then it's OK to leave it as is for now.

A new issue could be opened to handle this.